Sources/MapboxCoreNavigation/EHorizon/RoadGraph.swift

```diff
@@ -32,6 +32,35 @@ public final class RoadGraph {
         }
         return LineString(locations.map { $0.coordinate })
     }
+    
+    /**
+     Returns a line string geometry corresponding to the given path.
+     
+     - returns: A line string corresponding to the given path, or `nil` if any of path edges is inaccessible.
+     */
+    public func pathShape(for path: Path) -> LineString? {
```

Comment:
“Path” seems redundant in this method signature. Since a `RoadGraph.Path` is already being passed in, it can be just `shape(of:)`.